catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -388,7 +393,9 @@ private SolrDocumentList handleSpellcheck(
         responseProps.put(SHOWING_RESULTS_FOR_KEY, new ArrayList<>(corrections));
       }
     }
+    LOGGER.trace("End solr spellcheck/Starting highlight extraction: query request {}", request);
```

Comment:
❓ Similar to the above question: perhaps this should be separated into 2 log statements where the "end solr spellcheck" portion is inside the if statement above so that it only gets logged when performing spellcheck?  Then this line just represents the start of highlighting.

Reply:
done, and I did the same for highlighting, which just required adding a if-statement.